package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "dependencies": {
+    "standard": "^16.0.3"
+  },
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "standard",
+    "lint:fix": "standard --fix"
+  },
+  "standard": {
+    "ignore": [
+      "out/*"
+    ],
+    "env": [
+      "es6",
```

Comment:
JS here will be all ES5 me thinks

Reply:
hmm, no es5 option for standard, I'll remove it.